### PR TITLE
SCT-87

### PR DESCRIPTION
**Pull request by DawsonDease**

### CODE REVIEW

File: scutl.py [hunk 1]
```diff
@@ -919,27 +927,35 @@ def member_joined_channel(client, event, logger):
...
         if user_id in users:
             return
 
         # does this put us over the user allotment?
-        over_tier_limit = False
-        if org['tier'] == 'FREE':
-            if len(users) >= 25:
-                over_tier_limit = True
-
+        over_tier_limit = check_user_allotment(users, org)
```
Comment (JaycobDuffel) [inline scutl.py:952]: Nice job refactoring this to a reusable function!

**Review by JaycobDuffel — APPROVED** [review]

File: scutl.py (hunk 1 above)
Comment (DawsonDease) [reply scutl.py:952]: Thank you!

File: scutl.py [hunk 2]
```diff
@@ -919,27 +927,35 @@ def member_joined_channel(client, event, logger):
             )
             client.conversations_leave(channel=channel_id)
         else:
+            users = users_in_channel['members'] + list(set(users) - set(users_in_channel['members']))
+            over_tier_limit = check_user_allotment(users, org)
```
Comment (katiemcgoff) [inline scutl.py:931]: This is really interesting code! I'm not sure I've ever used set() before. Looks good.

One thing - I believe the metl bot counts as a user in the new channel, so you should remove it before you check the user count.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
One change re. the bot user being counted in the user count.

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**